src/core/execute.c

```diff
@@ -201,6 +202,63 @@ static const char *exec_context_tty_path(const ExecContext *context) {
         return "/dev/console";
 }
 
+static int exec_context_tty_size(const ExecContext *context, unsigned *ret_rows, unsigned *ret_cols) {
+        unsigned rows = UINT_MAX, cols = UINT_MAX;
+        const char *tty;
+        int r;
+
+        assert(context);
+        assert(ret_rows);
+        assert(ret_cols);
+
+        tty = skip_dev_prefix(exec_context_tty_path(context));
```

Comment:
`exec_context_tty_path()` may return `NULL`. Please add assertion about that it returns non-NULL, or return earlier if `NULL` is returned.